Add information on coexistence (#106670)
Azure/azure-content: articles/storage/files/storage-files-identity-auth-azure-active-directory-enable.md

Changes are not instant, and require a policy refresh or a reboot to take effect.

> [!NOTE]

> > Once this change is applied, the client will not be able to connect to Storage Accounts using on-premises AD DS integration without configuring Kerberos realm mappings.

Reviewer: ```suggestion
> Once this change is applied, the client will not be able to connect to storage accounts using on-premises AD DS integration without configuring Kerberos realm mappings.
```

> ### Configure coexistance with Storage Accounts using on-premises AD DS

Reviewer: ```suggestion
### Configure coexistence with storage accounts using on-premises AD DS
```

Add an entry for each storage account that uses on-premises AD DS integration. Use one of the following three methods:
- Configure this Intune Policy CSP and apply it to the client(s): [Kerberos/HostToRealm](/windows/client-management/mdm/policy-csp-admx-kerberos#hosttorealm)

> - Configure this group policy on the client(s): `Administrative Tempalte\System\Kerberos\Define host name-to-Kerberos realm mappings`

Reviewer: ```suggestion
- Configure this group policy on the client(s): `Administrative Template\System\Kerberos\Define host name-to-Kerberos realm mappings`
```

- Configure the following registry value on the client(s): `reg add HKLM\SYSTEM\CurrentControlSet\Control\Lsa\Kerberos\domain_realm /v <DomainName> /d <StorageAccountEndPoint>`

>     - For example, `reg add HKLM\SYSTEM\CurrentControlSet\Control\Lsa\Kerberos\domain_realm /v contoso.local /d staddsauth.file.core.windows.net`

Reviewer: ```suggestion
    - For example, `reg add HKLM\SYSTEM\CurrentControlSet\Control\Lsa\Kerberos\domain_realm /v contoso.local /d your-storage-account-name.file.core.windows.net`
```
